packages/ember-glimmer/tests/integration/components/fragment-components-test.js

```diff
@@ -165,6 +165,19 @@ moduleFor('Components test: fragment components', class extends RenderingTest {
     this.assertText('baz');
   }
 
+  ['@test does not throw an error if `tagName` is an empty string and `id` is bound property specified via template']() {
+    let template = `{{id}}`;
+    let FooBarComponent = Component.extend({
+      tagName: ''
+    });
+
+    this.registerComponent('foo-bar', { ComponentClass: FooBarComponent, template });
+
+    this.render(`{{#foo-bar id=fooBarId}}{{/foo-bar}}`, { fooBarId: 'baz' });
+
+    this.assertText('baz');
```

Comment:
Can you test updating the id and resetting it back to the original value? (The point of this test is to make sure we don't throw when you update the id value on a tag less component). :smile: